deploy/kubernetes/base/csidriver.yaml

```diff
@@ -0,0 +1,8 @@
+---
+
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: fsx.openzfs.csi.aws.com
```

Comment:
note: for file cache we have the metadata name as `filecache.csi.aws.com`, not sure if you have any thoughts on making it `fsxopenzfs.csi.aws.com` but wanted to bring it up

Reply:
hmm I'd prefer to keep it as fsx.openzfs.csi.aws.com, it looks a lot better imo